CRM/Utils/System/Base.php

```diff
@@ -705,7 +705,7 @@ public function getTimeZoneOffset() {
 
       $timeZoneOffset = sprintf("%02d:%02d", $tz / 3600, abs(($tz / 60) % 60));
 
-      if ($timeZoneOffset > 0) {
+      if ($tz >= 0) { // $tz is a number, not a string, so it is better to test it.  00:00 needs a plus sign.
```

Comment:
I can accept that the $tz value is better to test than timeZoneOffset as you point out. However, if it is 0 we will already have returned a few lines earlier.

Putting comments at the end of the line will trigger a style warning when we finally get the tests to run.

(NB if this is just tidy up rather than function you might leave it out because it will speed up QA if we only have to test on Joomla!)